content/millennium/dstu2/care-provision/goal.md

```diff
@@ -40,18 +46,18 @@ Search for Goals that meet supplied query parameters:
 
 ### Parameters
 
- Name              | Required?                                | Type          | Description
--------------------|------------------------------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- `_id`             | This, or one of `patient`, or `subject`. | [`token`]     | The logical resource id associated with the Goal. Example: `_id=7891`
- `patient`         | This, or one of `_id`, or `subject`.     | [`reference`] | The patient who has the goal. Example: `patient=12345`
- `subject:Patient` | This, or one of `_id`, or `patient`.     | [`reference`] | The subject who this goal is intended for. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/12345` or `subject:Patient=12345`
- `targetdate`      | N                                        | [`date`]      | A date or date range from which to find Goals. Example: `targetdate=ge2016-10-01&targetdate=le2016-12-01`
+ Name              | Required?        | Type          | Description
+-------------------|------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`             | Conditionally    | [`token`]     | The logical resource ID associated with the goal. This parameter is required if `patient` or `subject` are not used. Example: `_id=7891`
+ `patient`         | Conditionally    | [`reference`] | The patient who has the goal. This parameter is required if `_id` or `subject` are not used. Example: `patient=12345`
+ `subject:Patient` | Condtionally     | [`reference`] | The subject who this goal is intended for. Must represent a Patient resource. May use the `:Patient` modifier. This parameter is required if `_id` or `patient` are not used. Example: `subject=Patient/12345` or `subject:Patient=12345`
```

Comment:
we should remove the modifier on the field name, since it is not required and is included in the description and examples.

also, pluralize "Example:"